src/Topologies/topology2d.jl

```diff
@@ -119,61 +137,10 @@ boundary_tag(topology::Topology2D, boundary_name::Symbol) =
 
 boundary_faces(topology::Topology2D, boundary) = topology.boundaries[boundary]
 
-function neighboring_elements(topology::Topology2D, elem)
-
-    # Each interior elem in a Topology2D will have 8 neighboring elements
-    #
-    #    o------o------o------o
-    #    |      |      |      |
-    #    |      |      |      |
-    #    o------o------o------o
-    #    |      |      |      |
-    #    |      |      |      |
-    #    o------o------o------o
-    #    |      |      |      |
-    #    |      |      |      |
-    #    o------o------o------o
-    #
```

Comment:
I thought the ASCII art could have been useful, but it's ok if not. 

@charleskawczynski today when we were chatting about the number of neighboring elements, pulling this up could have helped picturing that

Reply:
I'm not sure it makes much sense to keep: the code no longer iterates over the various connections, instead it precomputes the neighbors for each element and iterates directly over those.

Reply:
If you think it would be suitable to move elsewhere, then do feel free.

Reply:
Absolutely, not a problem if that is not reflecting/helping explaining the current code. I was mentioning that because today Charlie and I were chatting about this neighboring elements search